test/integration/crud/crud_api.test.ts

```diff
@@ -1093,32 +1093,24 @@ describe('CRUD API', function () {
     }
   });
 
-  it('should correctly throw error on illegal callback when unordered bulkWrite encounters error', {
-    // Add a tag that our runner can trigger on
-    // in this case we are setting that node needs to be higher than 0.10.X to run
-    metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
-    },
-
-    test: async function () {
-      const ops = [];
-      // Create a set of operations that go over the 1000 limit causing two messages
-      let i = 0;
-      for (; i < 1005; i++) {
-        ops.push({ insertOne: { _id: i, a: i } });
-      }
+  it('should correctly throw error on illegal callback when unordered bulkWrite encounters error', async function () {
```

Comment:
drive by cleanup as I fixed a bug caught here. Nothing functionally changed.

Reply:
Can we fix the test description?  I think this is testing that when a multi-batch unordered bulk write encounters an error, we handle the error properly and return an error to the user?

Reply:
Updated the description, also could not resist improving the assertions a bit too to match the title.